src/c++/perf_analyzer/data_loader.h

```diff
@@ -36,6 +36,15 @@ namespace triton { namespace perfanalyzer {
 class NaggyMockDataLoader;
 #endif
 
+/// Data for one input or output tensor
+///
+struct TensorData {
```

Comment:
Is this the right place for this? I originally called it DataLoaderData which is why it was here. However, it is now used in infer_data_manager* as well as infer_data.h

Reply:
How about its own file?

Reply:
+1